.github/actions/linting/action.yml

```diff
@@ -9,34 +9,26 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: hashicorp/setup-terraform@v2
-      with:
-        terraform_version: ">=1.5.0"
+    - name: grept
```

Comment:
@matt-FFFFFF I think if we add `grept` now we might meet a chicken-egg dilemma, since we haven't finished these actions, the grept check won't success. I suggest maybe we can comment this step for now, once we've finished all jobs including post-push event and how to run `grept` weekly, and how grept job submit pr, then we can uncomment this check. WDYT?

Reply:
Ok I agree